client/state/purchases/reducer.js

```diff
@@ -112,7 +112,7 @@ function updatePurchases( existingPurchases, action ) {
 
 const assignError = ( state, action ) => ( { ...state, error: action.error } );
 
-export const items = createReducer( initialState, {
+export default createReducer( initialState, {
```

Comment:
If we look at other reducers we can see that items is usually used for the list of items (often called items, data or list). Each `reducer.js` module actually export the whole reducers containing the list and other meta information such as error, is Requesting...
Here items was actually the whole reducer (with the list and meta props) so we should export it as the default reducer for this module.
Check out how reducers are combines together here:
https://github.com/Automattic/wp-calypso/blob/master/client/state/index.js#L8